stats/stats.go

```diff
@@ -59,6 +59,27 @@ func (s *Begin) IsClient() bool { return s.Client }
 
 func (s *Begin) isRPCStats() {}
 
+// InitialResolverResult represents an event that the resolved finished
+// resolving if the ClientConn blocks on resolver resolution while performing a
```

Comment:
"...if the RPC occurred before the initial resolver result."?

Reply:
My string is still technically correct, but yours falls under the subset of my language and is more specific :). Switched.

Reply:
This is an external docstring, so we should find something that is as clear and concise as possible.

"blocks on resolver resolution" is confusing to read and technically the `ClientConn` doesn't block -- the RPC does.

Reply:
Right, I agree. Will push once I finish tests.